pagopa/io-app
# fix: [PE-638] Remove CGN card from wallet screen when unsubscribed

## Short description
This PR removes the CGN card from the wallet screen page when the user unsubscribes that functionality from the CGN details page

## List of changes proposed in this pull request
- Added `walletRemoveCardByType` from the cgnUnsubscriptionHandler in order to remove the CGN card from the wallet screen when the subscription is successful

## How to test
- With the dev-api-server started, onboard a new CGN card from the Wallet homepage CTA.
- Then, when you're inside the CGN details card, press on the CTA to unsubscribe.
- After that, you should be able not to see the CGN card from the wallet page.

## Preview
https://github.com/user-attachments/assets/0f3a21b0-2b4a-492f-80fc-3046c7e1c44a
